pypowsybl/network.py

```diff
@@ -161,6 +164,46 @@ def create_substations_data_frame(self) -> pd.DataFrame:
     def create_hvdc_lines_data_frame(self) -> pd.DataFrame:
         return self.create_elements_data_frame(_pypowsybl.ElementType.HVDC_LINE)
 
+    def create_switches_data_frame(self) -> pd.DataFrame:
+        return self.create_elements_data_frame(_pypowsybl.ElementType.SWITCH)
+
+    def update_elements_with_data_frame(self, element_type: _pypowsybl.ElementType, df: pd.DataFrame):
+        for seriesName in df.columns.values:
+            series = df[seriesName]
+            if seriesName.endswith('p0') or seriesName.endswith('q0') or seriesName.endswith('setpoint'):
```

Comment:
We have to stay generic there is no way to guess real type for use input. We have to get column types from Java and then create the dataframe. So @yichen88 could you add to the other PR a 
_powsybl.get_series_types(element_type) to help create the dataframe with the correct type?